MAINTAINERS

```diff
@@ -0,0 +1,16 @@
+# Official list of OpenEBS Maintainers.
+#
+# Names added to this file should be in the following format:
+#     Individual's name,@githubhandle, Company Name
+#
+# Please keep the below list sorted in ascending order.
+#
+#Maintainers
+"Kiran Mova",@kmova,MayaData
+"Vishnu Itta",@vishnuitta,MayaData
+"Mayank Patel",@mynktl,MayaData
```

Comment:
IMO name should be in alphabatical order ( we can check k8s , docker and other projects for ref)

Reply:
done